handlers.go

```diff
@@ -13,11 +13,14 @@ import (
 	"strings"
 	"sync"
 	"time"
+	"strconv"
+	"math"
 
 	"github.com/genuinetools/reg/clair"
 	"github.com/genuinetools/reg/registry"
 	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
+	"github.com/knopka/go-pagination-bootstrap"
```

Comment:
Not sure about this import. I'm not sure having an external import that renders HTML that is from outside this repository is the way to go, but I might be wrong.